hpx/parallel/datapar/execution_policy.hpp

```diff
@@ -26,17 +27,18 @@
 namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v1)
 {
     ///////////////////////////////////////////////////////////////////////////
-    /// Extension: The class datapar_task_execution_policy is an execution
+    /// Extension: The class sequential_task_execution_policy is an execution
```

Comment:
why the change from `datapar_task_execution_policy`? Shouldn't it be `dataseq_task_execution_policy`?